src/commands/voice/volume.ts

```diff
@@ -0,0 +1,41 @@
+import { PermissionLevels } from './../../types/commands.ts';
+import { bot } from "../../../cache.ts";
+import { createCommand, createSubcommand } from "../../utils/helpers.ts";
+
+createCommand({
+  name: "volume",
+  description: "See the current volume.",
+  aliases: ["vol", "v"],
+  arguments: [
+    { name: "subcommand", type: "subcommand", required: false }
+  ],
+  guildOnly: true,
+  execute(message) {
+    const player = bot.lavadenoManager.players.get(message.guildId.toString());
+
+    if (!player) return message.reply("No player in this guild.");
+    
+    return message.reply(`Current volume is ${player?.volume}.`);
```

Comment:
seems it can be undefined so add a fallback?

Reply:
`.volume` cannot be undefined as long as player exists.
So just `player.volume` would do.

Reply:
I don't really know if this can be undefined. VSCode just autocompleted that for me lol

Reply:
It cannot, it defaults to `100` on creation.